circuit/circuit_js/witness_calculator.js

```diff
@@ -149,7 +149,7 @@ class WitnessCalculator {
 		throw new Error(`Too many values for input signal ${k}\n`);
 	    }
             for (let i=0; i<fArr.length; i++) {
-                const arrFr = toArray32(normalize(fArr[i],this.prime),this.n32)
+                const arrFr = toArray32(BigInt(fArr[i])%this.prime,this.n32)
```

Comment:
Why edit this?

Reply:
didn't make any edits here, auto generated?